### ISSUE TITLE
3.0 - Standardize Form Controls in the Downloads Metaboxes

### ISSUE THREAD

**Issue by robincornett** [easydigitaldownloads/easy-digital-downloads#7742]
## Enhancement Request
Update the download metabox form fields to have associated labels.

### Justification or use case
Adding proper labels to all form inputs will improve both semantics and accessibility. Running just an automated accessibility scan on a download notes 12 form elements without labels, some of which are duplicates in repeatable rows. There also are some labels which are more accurately descriptive text which should be checked and updated.

Related: #7739
